base/ltmath.dtx

```diff
@@ -754,6 +754,45 @@
 % \end{macro}
 % \end{macro}
 %
+%
+%
+%  \begin{macro}{\dollardollar@begin,\dollardollar@end}
+%    Two commands for starting and ending display math formulas. They
+%    are needed for tagging support and should be used by all display
+%    math environments if they want to be supported by tagging. We
+%    pretend that they have been always available, so that rollback
+%    doesn't break packages that have not set up their own rollback.
+% \changes{v1.2o}{2025/03/08}{\cs{dollardollar@begin} and
+%                 \cs{dollardollar@end} introduced for tagging}
+%    \begin{macrocode}
+%</2ekernel>
+%<*2ekernel|latexrelease>
+%<latexrelease>\IncludeInRelease{2025/06/01}%
+%<latexrelease>  {\dollardollar@begin}{Start and end display math}%
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\def\dollardollar@begin{$$}
+\def\dollardollar@end{$$}
```

Comment:
Is it good for these two names to be long/complex?
Or would much shorter names be better?

Reply:
They are needed in perhaps two dozen places across all of TeX Live, so I don't see why making them shorter would be useful. I don't mind different names if anybody has a good suggestion, but this here seems to me fairly clear and understandable to a package programmer.

Reply:
the names are OK or you could go for a name based on usage rather than the definition say `\mathdisplay@begin` `\mathdisplay@end` but really they have to be `$$` not any morecomplec code you might want to start or end a display, so I think the current names are OK

Reply:
But note that you are now making these long names available and encouraged for further use in packages/classes: which again raises the importance of more manageable (i.e., shorter and clearer) names and good documentation on where and how they should, or must, be used.

Reply:
If you have a suggestion send it in for discussion -- this can easily be changed between now and June. But as I said I think they are right the way they are now and clearly indicate what they are supposed to indicate, i.e., "use `\dollardollar@begin` instead of `$$` if you start a TeX display in your code and use `\dollardollar@end` when you end it".

Reply:
so what? We did know that from the start; they are used in a few packages and classes that set up special math environment. Again, if you have a name suggestion, let's have it. But keep in mind that short isn't good as such it only hightens the likelyhood that the suggested name is already used.

Reply:
Not that it matters much, but some suggestions anyway:

\dollars@begin/end

\ddollars@begin/end

\begin/end@dollsrs

etc.